python/setup.py

```diff
@@ -23,6 +23,9 @@
 include_dirs = [os.path.dirname(sysconfig.get_path("include")),]
 library_dirs = [get_python_lib()]
 
+if nvtx_dir := os.getenv("NVTX_DIR"):
+    include_dirs.insert(0, os.path.join(nvtx_dir, "include"))
```

Comment:
Actually for most people this will be CUDA_PATH on windows. Did you on purpose name it differently ? 
`C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v11.8\include\nvtx3`

Reply:
I am also fine doing that ! 👍

Reply:
Where is this env var supposed to be set?  Also, I generally don't like to set the include path to .../include/nvtx3, but rather just to .../include, and make the source files do `#include <nvtx3/nvToolsExt.h>` so they don't inadvertently pick up the old version of nvToolsExt.h from the CUDA include directory (where cuda.h is).